src/components/accordion-item/daikin-accordion-item.ts

```diff
@@ -98,7 +80,27 @@ export class DaikinAccordionItem extends LitElement {
 
     :host {
       display: block;
+      position: relative;
+    }
+
+    :host::before {
+      content: "";
+      display: none;
```

Comment:
Maybe we should use CSS variable for clarity:

```suggestion
      display: var(--divider-top-display, none);
```